apps/web/lib/app-providers-app-dir.tsx

```diff
@@ -273,7 +274,15 @@ const AppProviders = (props: PageWrapperProps) => {
                 isBookingPage={props.isBookingPage || isBookingPage}>
                 <FeatureFlagsProvider>
                   <OrgBrandProvider>
-                    <MetaProvider>{props.children}</MetaProvider>
+                    <MetaProvider>
+                      {props.children}
+                      <DubAnalytics
+                        cookieOptions={{
+                          // here we're using NEXTAUTH_COOKIE_DOMAIN for cross-domain tracking
+                          domain: process.env.NEXTAUTH_COOKIE_DOMAIN,
```

Comment:
This env variable is not available client-side. Are we ok with this?

Reply:
Good catch – I'll get this updated 👍

Reply:
Ended up hard-coding the domain but limiting this to only show when `IS_CALCOM` – LMK if that's okay!